lib/wallaroo/core/topology/runner.pony

```diff
@@ -717,6 +721,7 @@ class StateRunner[S: State ref] is (Runner & ReplayableRunner &
       (is_finished, last_ts)
     else
       @printf[I32]("StateRunner: Input was not a StateProcessor!\n".cstring())
+      Fail()
       (true, latest_ts)
```

Comment:
Are the return tuples needed? Let's remove them as to not confuse our future selves.

Reply:
Yes the are needed.

Reply:
Without those return value would be the return type from Fail() and that wouldnt type check.